.vscode/settings.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
how come this is being deleted?

Reply:
This was a mistake. It should not be gitignored or all of the extensions settings will be broken and will not work.

We've also lost the recommended extensions somewhere. I'm not sure why the `.vscode` was added to the gitignore in an unrelated PR months ago.

I'll add this back to get things working again.